# UAV Task Planner Environment Configuration

waypoints:
  landing_pad: [0.0, 0.0, 0.0]
  waypoint_a: [1.0, 2.0, 1.0]
  waypoint_b: [3.0, 4.0, -1.0] 
  waypoint_c: [0.0, 1.0, 2.0]

# Required by config loader but not used by executor
flight_zone:
  min: [-5.0, -5.0, 0.3]
  max: [5.0, 5.0, 3.0]

safety:
  min_altitude: 0.3
  max_altitude: 3.0

drone:
  start_position: [0.0, 0.0, 0.0]
  max_speed: 2.0
  battery_capacity: 100